src/test/resources/net/sf/jabref/importer/fileformat/InspecSilverPlatterImporterTest.txt

```diff
@@ -1,30 +0,0 @@
-Record INSPEC
```

Comment:
I thought that Inspec and SilverPlatter are two different formats. Therefore, I would name the file accordingly. - Maybe `SilverPaltterGeneralFile.txt`